config/rbac/appstudio_api_permissions_role.yaml

```diff
@@ -0,0 +1,36 @@
+# permissions to scale Camel K resources.
```

Comment:
is this up to date with the actual purpose of this role?

Reply:
Good catch. bad copy-paste - fixed in f8fbda0